README.md

```diff
@@ -0,0 +1,18 @@
+# sinol-make
```

Comment:
Currently the project is not ready to be used, thus it might be best to inform about this in the `README.md` and update it when we'll have clear installation steps.

Reply:
ebc5c617e28d00469f54f68baab4408506c8c5fe